server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraSessionModule.java

```diff
@@ -76,12 +76,14 @@ Cluster provideCluster(FileSystem fileSystem, AsyncRetryExecutor executor) throw
         PropertiesConfiguration configuration = getConfiguration(fileSystem);
 
         return getRetryer(executor, configuration)
-                .getWithRetry(ctx -> ClusterWithKeyspaceCreatedFactory.clusterWithInitializedKeyspace(
-                        ClusterFactory.createClusterForSingleServerWithoutPassWord(
-                                configuration.getString("cassandra.ip"),
-                                configuration.getInt("cassandra.port")),
-                        configuration.getString("cassandra.keyspace"),
-                        configuration.getInt("cassandra.replication.factor")))
+                .getWithRetry(ctx -> ClusterWithKeyspaceCreatedFactory
+                        .config(
+                            ClusterFactory.createClusterForSingleServerWithoutPassWord(
+                                    configuration.getString("cassandra.ip"),
+                                    configuration.getInt("cassandra.port")),
+                            configuration.getString("cassandra.keyspace"))
+                        .replicationFactor(configuration.getInt("cassandra.replication.factor"))
```

Comment:
I could be cool to expose durable writes here. You could then overright the default "false" value in the test in order to take advantage of this speed up for jmap-integration testing.
What do you think ?